pa.sh

```diff
@@ -3,7 +3,10 @@
 export PASH_TOP=${PASH_TOP:-${BASH_SOURCE%/*}}
 export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/usr/local/lib/"
 # point to the local downloaded folders
+export PYTHONPATH=$PASH_TOP/venv/lib/python3.12/site-packages:$PYTHONPATH
 export PYTHONPATH="${PASH_TOP}/python_pkgs/:${PYTHONPATH}"
+export PYTHONPATH=~/annotations:$PYTHONPATH
```

Comment:
This is too hardcoded. Maybe the default value is next to the pash directory (sibling of PaSh top).